extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -597,7 +593,7 @@ export class ApplicationInsights extends BaseTelemetryPlugin implements IAppInsi
             // We want to enable exception auto collection and it has not been done so yet
             const onunhandledrejection = "onunhandledrejection";
             const originalOnUnhandledRejection = _window[onunhandledrejection];
-            _window.onunhandledrejection = (error: PromiseRejectionEvent) => {
+            _window[onunhandledrejection] = (error: PromiseRejectionEvent) => {
```

Comment:
Changing to index assignment to avoid the TS error and this also means we don't need the additional declaration to add this for older TS versions.